include/mls/credential.h

```diff
@@ -60,6 +60,9 @@ struct X509Credential
 
 private:
   SignaturePublicKey _public_key;
+  hpke::Signature::ID _signature_algorithm;
```

Comment:
This exposes the HPKE library, which I would rather not do.  But on the other hand, hiding it involves some contortions.  Not sure what the best plan is here.